In addition to dynamically linking with the FFmpeg libraries, Tenacity wants to
be able to dlopen them at runtime. Fix the path so that it can find them where
MacPorts installed them. The Portfile will replace the @PREFIX@ placeholder with
the correct value.

https://codeberg.org/tenacityteam/tenacity/issues/401
--- libraries/lib-ffmpeg-support/FFmpegFunctions.cpp.orig
+++ libraries/lib-ffmpeg-support/FFmpegFunctions.cpp
@@ -328,8 +328,7 @@ std::vector<wxString> FFmpegFunctions::GetSearchPaths(bool fromUserPathOnly)
       paths.emplace_back(path);
 
 #elif defined(__WXMAC__)
-   paths.emplace_back(wxT("/Library/Application Support/tenacity/libs"));
-   paths.emplace_back(wxT("/usr/local/lib/tenacity"));
+   paths.emplace_back(wxT("@PREFIX@/lib"));
 #endif
 
    return paths;
